packages/notifications/src/styled/content/StyledClose.ts

```diff
@@ -5,71 +5,79 @@
  * found at http://www.apache.org/licenses/LICENSE-2.0.
  */
 
-import styled from 'styled-components';
-import {
-  retrieveComponentStyles,
-  getColorV8,
-  DEFAULT_THEME,
-  focusStyles
-} from '@zendeskgarden/react-theming';
-import { Hue } from '../../utils/useNotificationsContext';
+import styled, { DefaultTheme, ThemeProps, css } from 'styled-components';
+import { retrieveComponentStyles, DEFAULT_THEME, getColor } from '@zendeskgarden/react-theming';
+import { Type } from '../../types';
+import { validationTypes } from '../../utils/icons';
+import { IconButton } from '@zendeskgarden/react-buttons';
 
 const COMPONENT_ID = 'notifications.close';
 
 interface IStyledCloseProps {
-  hue?: Hue;
+  $type?: Type;
 }
 
+/**
+ * 1. IconButton reset
+ */
+const colorStyles = ({ theme, $type }: IStyledCloseProps & ThemeProps<DefaultTheme>) => {
+  let variable;
+  let color;
+  let hoverColor;
+  let activeColor;
+
+  if ($type) {
+    switch ($type) {
+      case validationTypes.warning:
+        variable = 'foreground.warning';
+        break;
+      case validationTypes.error:
+        variable = 'foreground.danger';
+        break;
+      case validationTypes.success:
+        variable = 'foreground.success';
+        break;
+      default:
+        variable = 'foreground.subtle';
+        break;
+    }
+
+    color = getColor({ variable, theme });
+    hoverColor = getColor({ variable, light: { offset: 100 }, dark: { offset: -100 }, theme });
+    activeColor = getColor({ variable, light: { offset: 200 }, dark: { offset: -200 }, theme });
+  }
```

Comment:
This is related to this [comment](https://github.com/zendeskgarden/react-components/pull/1842/files#r1649101016). I think we should let the `default` case handle a missing `$type` otherwise we're returning CSS with `undefined` colors. In other words, removing the `if($type)` statement.

```suggestion
switch ($type) {
  case validationTypes.warning:
    variable = 'foreground.warning';
    break;
  case validationTypes.error:
    variable = 'foreground.danger';
    break;
  case validationTypes.success:
    variable = 'foreground.success';
    break;
  default:
    variable = 'foreground.subtle';
    break;
}

color = getColor({ variable, theme });
hoverColor = getColor({ variable, light: { offset: 100 }, dark: { offset: -100 }, theme });
activeColor = getColor({ variable, light: { offset: 200 }, dark: { offset: -200 }, theme });

```

Reply:
Ah, yes, good call out!